rosys/hardware/bumper.py

```diff
@@ -50,16 +50,17 @@ def __init__(self, robot_brain: RobotBrain, *,
                          estop=estop)
 
     def handle_core_output(self, time: float, words: list[str]) -> None:
+        active_bumpers = list(self.active_bumpers)
```

Comment:
Why wrapping with `list()`?
And maybe `previous_active_bumpers` is a better name?